examples/simple-nginx.md

```diff
@@ -6,10 +6,9 @@ to Kubernetes and running your first containers on the cluster.
 
 ### Running a container (simple version)
 
-From this point onwards, it is assumed that ```kubectl``` is on your path from one of the getting started guides.
+From this point onwards, it is assumed that `kubectl` is on your path from one of the getting started guides.
 
-The `kubectl` line below spins up two containers running
-[Nginx](http://nginx.org/en/) running on port 80:
+The `kubectl` line below first specifies the name of the [repilcation controller](/docs/replication-controller.md) to create `my-nginx`. Then, the [Docker image](https://registry.hub.docker.com/_/nginx/) to run `--image=nginx`. Next, the number of [pod](/docs/pods.md) replicas to create `--replicas=2`. Finally, the port to open for each pod `--port=80`.
```

Comment:
Can we instead add a brief description on what run does - creates a replication controller with specified number of replica pods. Most args are self-evident. We can just add a pointer to https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl_run.md